index.html

```diff
@@ -5,217 +5,219 @@
-        </p>
-         <ul class="lang">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-         </ul>
-         <button type="button">See Project</button>
-        </div>
-    
     </section>
 
-    <section>
-    
-    <div class="about">
-        <h2>About </h2>
-        <h3> MySelf</h3>
-        <p class="intro">
-            Hello I am a software developer! I can help you build a product ,feature or website Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <div class="contact">
-            <h4>LET'S  CONNECT</h4>
-             </div>
-             <div class="icons">
-        <ul>
-          <li><img src="image/image6.png" alt="twitter"></li>
-          <li><img src="image/image7.png" alt="twitter"></li>
-          <li><img src="image/image8.png" alt="twitter"></li>
-           <li><img src="image/image9.png" alt="twitter"></li>
-          <li><img src="image/imge5.png" alt="twitter"></li>
-        </ul>
-        </div>
-          
-          <button type="button">Get My Resume</button>
+    <section class="about">
+        <div class="about-card">
+        <div class="introduction-2">
+            <h1 class="title">About <br>
+            Myself</h1>
+           <p class="paragraph">I’m a fullstack web developer! I can help you build a <br> website, feature or website Look through some of my<br> work and experience! If you like what you see and have <br> a project you need coded, don’t hestiate to contact me.</p>
+           <h2 class="connect">Let's Connect</h2> 
+           <ul class="icons">
+             <li><img src="imgs/social media icons (1).png" alt="social media icons"></li>
+            </ul>
+            <button class="ME" type="button">Get My Resume</button>
+          </div>  
 
-                
-                <div class="end">
-                 <ul class="photo">
-                <li><h5>Languages</h5><img src="/image/imge8.png" alt="photo"></li>
+        <div class="languages">
+            <ul class="photo">
+                <li><h5>Languages</h5><img src="/imgs/imge8.png" alt="photo"></li>
                 </ul>
                         <div class="fitems">
-                             <ul>
-                            <li><img src="/image/Ellipse1.png" alt="1">Javascript</li>
-                           </ul>
-                            <ul>
-                            <li><img src="/image/Ellipse2.png" alt="2">CSS</li>
-                            </ul>
-                             <ul>
-                            <li><img src="/image/ellipse3.png" alt="3">HTML</li>
-                            </ul>
-                          </div>
+                            <ul class="item-1">
+                               <li>
+                                <div class="java">
+                                <img src="/imgs/Ellipse0.png" alt="1">
```

Comment:
- [ ] Please 🙏🏽 review the path of the images for a better user experience.

| **Figma template** | **Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/193379799-3583e8f3-6ecb-43b7-8938-3cf1ca2c5aaa.png) | ![image](https://user-images.githubusercontent.com/58642949/193379808-fe222f78-4b97-4b7f-bce6-381c28baa6e0.png) |